contracts/pool/Pool.sol

```diff
@@ -251,47 +197,32 @@ contract Pool is PoolFDT {
 
     // Get LiquidityLocker balance
     function _balanceOfLiquidityLocker() internal view returns (uint256) {
-        return liquidityAsset.balanceOf(liquidityLocker);
-    }
-
-    // Checks that the current state of Pool matches the provided state
-    function _isValidState(State _state) internal view {
-        require(poolState == _state, "P:BAD_STATE");
-    }
-
-    // Emits a `BalanceUpdated` event for LiquidityLocker
-    function _emitBalanceUpdatedEvent() internal {
-        emit BalanceUpdated(liquidityLocker, address(liquidityAsset), _balanceOfLiquidityLocker());
+        return liquidityAsset.balanceOf(address(liquidityLocker));
     }
 
     // Checks that the protocol is not in a paused state
     function _whenProtocolNotPaused() internal view {
         require(!_globals(superFactory).protocolPaused(), "P:PROTO_PAUSED");
     }
 
-    // Checks that `msg.sender` is the Pool Delegate and not paused
-    function _isValidDelegateAndProtocolNotPaused() internal view {
-        require(msg.sender == poolDelegate, "P:NOT_DEL");
-        _whenProtocolNotPaused();
-    }
-
     // Transfers Liquidity Asset to given `to` address
-    function _transferLiquidityAssetFrom(address from, address to, uint256 value) internal {
+    function _transferLiquidityAssetFrom(
+        address from,
+        address to,
+        uint256 value
+    ) internal {
         liquidityAsset.safeTransferFrom(from, to, value);
     }
 
-    // Transfers Liquidity Locker assets to given `to` address
-    function _transferLiquidityLockerFunds(address to, uint256 value) internal returns (bool){
-        return _liquidityLocker().transfer(to, value);
-    }
-
     // Returns the LiquidityLocker instance
     function _liquidityLocker() internal view returns (ILiquidityLocker) {
         return ILiquidityLocker(liquidityLocker);
     }
 
     // Returns the HeliosGlobals instance
-    function _globals(address poolFactory) internal view returns (IHeliosGlobals) {
+    function _globals(
+        address poolFactory
+    ) internal view returns (IHeliosGlobals) {
         return IHeliosGlobals(IPoolFactory(poolFactory).globals());
     }
 
```

Comment:
if we remove borrower's role from the contract, we should probably get rid of  isBorrower() modifier